robottelo/hosts.py

```diff
@@ -71,6 +71,18 @@
 }
 
 
+@lru_cache(maxsize=128)
+def lru_sat_ready_rhel(rhel_ver):
+    deploy_args = {
+        'deploy_rhel_version': rhel_ver or '8',
```

Comment:
Yeah thats coming from the existing installer tests fixture implementation. 

I would insted use `settings.server.version.rhel_version` that would be change from CI for two different RHEL versions in future instead of `constants` value.